src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorToKfsVendorDetailConversionServiceImpl.java

```diff
@@ -63,7 +65,12 @@ protected KfsVendorDataWrapper createVendorDetail(PaymentWorksVendor pmwVendor,
                                                       Map<String, List<PaymentWorksIsoFipsCountryItem>> paymentWorksIsoToFipsCountryMap, 
                                                       Map<String, SupplierDiversity> paymentWorksToKfsDiversityMap) {
         
-        KfsVendorDataWrapper kfsVendorDataWrapper = populateTaxRuleDependentAttributes(pmwVendor, paymentWorksIsoToFipsCountryMap);
+        KfsVendorDataWrapper kfsVendorDataWrapper = null;
```

Comment:
should this be set to null?

Reply:
I removed the null check, and added a throw if there is an invalid form processing mode.